src/dataframe/dataframe.jl

```diff
@@ -1355,6 +1418,7 @@ function Base.push!(df::DataFrame, row::Any)
                      " $(typeof(row)) to be pushed into a DataFrame", :push!)
```

Comment:
rather say which will be allowed to be pushed (or refer to the docs) as this is probably more helpful than just saying what won't work anymore 😅

Reply:
fixed